resolwe_bio/docker_images/rnaseq/README.md

```diff
@@ -9,11 +9,14 @@ Included bioinformatics tools:
 * Cufflinks (v2.2.1)
 * Cutadapt (v1.14)
 * DESeq2 (v1.16.1)
+* DEXSeq (v1.24.2)
 * FastQC (v0.11.5)
 * HISAT2 (v2.1.0)
 * HTSeq (v0.9.1)
+* HTSeq (python2) (v0.6.1p1)
```

Comment:
Why do we need this?

Reply:
There are python2 scripts in DEXSeq and Subread_to_DEXSeq that use HTSeq. These scripts are part of the packages and should not be tempered with. We could ask the authors to change them or make pull requests to their git directories.